core/src/com/po/fuck/model/constants/ConstantsLoader.java

```diff
@@ -0,0 +1,32 @@
+package com.po.fuck.model.constants;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConstantsLoader {
+    protected static Properties loadProperties(String filePath) {
+        try {
+            Properties properties = new Properties();
+            properties.load(new FileInputStream(filePath));
+            return properties;
+        } catch (IOException io) {
+            throw new RuntimeException("Unable to load properties file: " + filePath + " - " + io.getMessage());
+        }
+    }
```

Comment:
Try to cache it, because you are re-reading the same file for each constant.

Reply:
I do not.